<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentA</class>
 <widget class="QWidget" name="studentA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加学生</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>请输入学生姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="snameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>请输入学生性别：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ssexEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLineEdit" name="monthEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>月</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLineEdit" name="dayEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>日</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>请输入学生专业：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="majorEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加学生</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>请输入学生学号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="snoEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>120</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入学生生日：</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="yearEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>年</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
